model.tex

```diff
@@ -855,6 +855,12 @@ \subsubsection{Location}
 \label{sec:orientation}
 The \sbol{orientation} property is OPTIONAL and has a data type of \sbol{URI}. All subclasses of \sbol{Location} share this property, which can be used to indicate how the region specified by the \sbol{SequenceAnnotation} and any associated double-stranded \sbol{Component} is oriented on the \sbol{elements} of a \sbol{Sequence}  from their parent \sbol{ComponentDefinition}. \ref{tbl:orientation_types} provides a list of REQUIRED \sbol{orientation} \sbol{URI}s. If a \sbol{Location} object has an \sbol{orientation}, then it MUST come from \ref{tbl:orientation_types}.
 
+\twothreezero{
+\paragraph{The \sbolheading{sequence} property}
+\label{sec:sequence}
+The \sbol{sequence} property is OPTIONAL and has a data type of \sbol{URI}. All subclasses of \sbol{Location} share this property, which indicates which \sbol{Sequence} object referenced by the containing \sbol{ComponentDefinition} is referenced by the \sbol{Location}.
```

Comment:
Isn't the data type `Sequence`? Technically it's a URI, but the way we refer to it in the ComponentDefinition portion of the spec if:

> The sequences property is OPTIONAL and MAY include a set of URIs that refer to Sequence objects.